data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/beaconhandlers/BeaconValidatorsHandler.java

```diff
@@ -65,22 +63,30 @@ public BeaconValidatorsHandler(
   @OpenApi(
       path = ROUTE,
       method = HttpMethod.GET,
-      summary = "Get validators from the running beacon node that match the specified query.",
+      summary = "Returns validators that match the specified query.",
       tags = {TAG_BEACON},
       description =
-          "Requests validator information. If no parameters specified, all current validators are returned.",
+          "Returns validator information. If no parameters specified, the first page of current validators are returned.",
```

Comment:
should probably say 'active' if that's what we're referring to by 'current'?

Reply:
no it's current as in "as of the current epoch"

Reply:
hopefully clearer now